internal/cli/opsmanager/logs/decrypt.go

```diff
@@ -42,12 +44,23 @@ func (opts *DecryptOpts) initFiles() func() error {
 	}
 }
 
+func (opts *DecryptOpts) stdOutMode() bool {
+	return opts.Out == ""
```

Comment:
```suggestion
	return opts.Out == "" || opts.Out == "/dev/stdout"
```
but this depends on the discussion about "/dev/stdout" (see below)